src/utils/index.js

```diff
@@ -15,6 +17,15 @@ export const createInQuery = (attribute, filters) => filters.length > 0
   ? [{selector: attribute, comparison: '=in=', arguments: filters}]
   : []
 
+export const createANDQuery = (attribute, filters) => {
+  if (filters.length > 0) {
+    let andQueryParts = []
+    for (let filter of filters) {
+      andQueryParts.push(`${attribute}==${filter}`)
+    }
+    return encodeRsqlValue(`(${andQueryParts.join(';')})`)
+  } else return ''
```

Comment:
Here, too, just create the abstract rsql syntax tree and let the library do the formatting and encoding